Library/Tracking/KSRAnalytics.swift

```diff
@@ -409,6 +420,7 @@ public final class KSRAnalytics {
       case .unwatch: return "unwatch"
       case .watch: return "watch"
       case .watched: return "watched"
+      case .pledge(let pledgeContext): return pledgeContext.trackingString
```

Comment:
Can you list these in alphabetical order? We generally try to do that wherever we can.